packages/amount-selectors/src/components/AmountSelectors.js

```diff
@@ -62,10 +72,11 @@ class AmountSelectors extends React.Component {
             min={this.props.min}
             max={this.props.max}
             step={this.props.step}
-            value={this.state.value}
+            value={this.state.displayValue}
```

Comment:
Couldn't we simply use `this.state.value.toFixed(2)` here instead of having this `displayValue` state?
Am I missing something?